README.md

```diff
@@ -11,12 +11,15 @@ parts:
         plugin: cmake
 ```
 
-And precede your `apps` entry like this:
+And append to your `apps.command-chain` entry like this:
 
 ```yaml
 apps:
     app-name:
-        command: bin/anon-shm-preload $SNAP/<binary>
+      command-chain:
+        // ...
+        - bin/anon-shm-preload
+    command: <app-binary>
```

Comment:
Indentation wrong, `command` should be under app. I also want to make this more compact